framework/basehandlers.py

```diff
@@ -217,23 +219,6 @@ def _update_last_visit_field(self, email):
     app_user.put()
     return True
 
-  def do_get(self, **kwargs):
```

Comment:
I'll do some exploration of this and make sure we have the same functionality as before if it's what we want.

Reply:
I made changes in [this commit](https://github.com/GoogleChrome/chromium-dashboard/pull/2324/commits/9bacd1ae58dc82d9e729de0ec0ae824d3ce7661f) that addresses this. The site should handle every case as it did previously while still confirming to mypy standards. 🙂

Reply:
I feel like we are losing too much guidance to subclass authors by not having these methods defined in the base class.   How about keeping these here and keeping `**kwargs` here, but changing the subclass methods to also use `**kwargs` instead of named parameters?

Reply:
Sure, I can give that a shot. Sorry, I know figuring out the approach to this part is a headache, and to be honest I'm not sure the best practice for a situation like this. The hope is that this change helps in the long run, so I want to find a solution that is as intuitive as possible :^)